packages/editor/src/store/utils/notice-builder.js

```diff
@@ -97,14 +97,19 @@ export function getNotificationArgumentsForSaveFail( data ) {
 	// If the post was being published, we show the corresponding publish error message
 	// Unless we publish an "updating failed" message
 	const messages = {
-		publish: __( 'Publishing failed' ),
-		private: __( 'Publishing failed' ),
-		future: __( 'Scheduling failed' ),
+		publish: __( 'Publishing failed.' ),
+		private: __( 'Publishing failed.' ),
+		future: __( 'Scheduling failed.' ),
 	};
-	const noticeMessage = ! isPublished && publishStatus.indexOf( edits.status ) !== -1 ?
+	let noticeMessage = ! isPublished && publishStatus.indexOf( edits.status ) !== -1 ?
 		messages[ edits.status ] :
-		__( 'Updating failed' );
+		__( 'Updating failed.' );
 
+	// Check if message string contains HTML. Notice text is currently only
+	// supported as plaintext, and stripping the tags may muddle the meaning.
+	if ( error.message && ! ( /<\/?[^>]*>/.test( error.message ) ) ) {
+		noticeMessage = sprintf( __( '%1$s Error message: %2$s' ), noticeMessage, error.message );
```

Comment:
This is missing a translator comment to explain the parameters.

Reply:
Should this be another PR?

Reply:
Yes